### ISSUE TITLE
Re-program the whole shader renderer

### ISSUE THREAD

**Issue by Lukevdb01** [Luke-Creative-Media/StellarEngine#21]
We need to make it possible that we can render object with just something like this: Pipeline::Draw("mesh_name"); it should then contain all the shader files etc

**Comment by jonesy-b-dev** [comment]
Thing for later for now, gotta make first implementation of lighting first, which is global a global shader.

Need to do lot of research to make per entity shader work i think

**Comment by Lukevdb01** [comment]
Yes, but we need to change the whole renderer for the entity system to let it work correct if it works properly we can then make light components

So I don't no if it is smart to wait for this and do lighting first.
